spec/csv_generator/csv_generator_spec.rb

```diff
@@ -94,8 +94,8 @@
         '',
         '',
         'false',
+        well1.position,
```

Comment:
What are these changes for? Were they in the wrong order before?

Reply:
Samples now need to have well position and I have removed sample names as it was no longer required.

Reply:
Ah got it thanks